### PR TITLE
Inkd button styling

### PR DESCRIPTION
**Pull request by tiagodcosta**

### CODE REVIEW

File: inkd-dev/inkd-button/inkd-button-styles.scss [hunk 1]
```diff
@@ -0,0 +1,24 @@
+@import '../global-styles/index';
+
+//@extend
+.button-red {
```
Comment (justin-c-rounds) [inline inkd-dev/inkd-button/inkd-button-styles.scss]: Should be a placeholder instead of a class.

File: package.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "inkd-ml",
+  "name": "inkd-button-styled",
```
Comment (justin-c-rounds) [inline package.json]: Why did this change?

Comment (tiagodcosta) [reply package.json]: It was by accident. I fixed in the next commit.

File: inkd-dev/inkd-button/inkd-button-styles.scss (hunk 1 above)
Comment (tiagodcosta) [reply inkd-dev/inkd-button/inkd-button-styles.scss]: Yes. I'm working on the idea of using mixins for buttons

File: .gitignore [hunk 3]
```diff
@@ -29,6 +29,7 @@ build/Release
 # Dependency directories
 node_modules
 jspm_packages
+bower_components
```
[the remark this answers is not in the document]
Comment (tiagodcosta) [reply .gitignore]: I will remove too. I added to avoid the problem when I ran bower on root.

**Comment by justin-c-rounds** [comment]
:shipit: Squash and merge

File: inkd-dev/global-styles/_variables.scss [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+//Sizes
+$small: .8rem;
+$medium: 1.1rem;
+$large: 1.4rem;
+$jumbo: 1.7rem;
+
+//Shapes
+$normal: 0.125em;
+$round: 50px;
```
Comment (justin-c-rounds) [inline inkd-dev/global-styles/_variables.scss]: Why is this `px` instead of an `em` value?

File: inkd-dev/inkd-button/inkd-button-styles.html [hunk 5]
```diff
@@ -0,0 +1,7 @@
+<dom-module id="inkd-button-styles">
+    <template>
+      <style>@import url("https://fonts.googleapis.com/css?family=Open+Sans");:host{all:initial;background:#3498db;color:#fff;font-size:1.1rem;font-family:inherit;visibility:inherit;border-radius:.125em;padding:.31em .62em;margin:.31em;transition:all 300ms ease;white-space:nowrap;cursor:pointer}:host(:hover:not([disabled])){background:#196090;color:#fff}:host(:active){background:#0a2639;color:#fff}:host([disabled]){opacity:0.5;box-shadow:none;cursor:not-allowed}
```
Comment (justin-c-rounds) [inline inkd-dev/inkd-button/inkd-button-styles.html]: Seems weird that the font import is here although the font is not being used.

Comment (tiagodcosta) [reply inkd-dev/inkd-button/inkd-button-styles.html]: Yes that's weird when working with components. I'll add some import conditional.

File: inkd-dev/global-styles/_variables.scss (hunk 4 above)
Comment (tiagodcosta) [reply inkd-dev/global-styles/_variables.scss]: I need to do some math ;-) $round: 3.125em;

**Pull request merged**